modules/effects/arrow-effects-data/src/main/kotlin/arrow/effects/typeclasses/suspended/prelude.kt

```diff
@@ -0,0 +1,5 @@
+package arrow.effects.typeclasses.suspended
```

Comment:
`prelude` and `predef` are common names in fp for the file with global static functions that are part of the base of the program for example `identity`. If anyone has a better suggestion for Kotlin it can be changed

Reply:
Right now it looks like there's just a callback inside. Maybe we can put that somewhere else or alternatively name the file to something more semantic related to what the callback is used for

Reply:
Prelude is not `Utils`. Prelude is what comes before something, the beginning and building blocks needed. in the global scope to compose programs. I renamed it for now to `Predef` so it's consistent with optics and Arrow core which is where the `identity` lives now. If we want a different strategy it would have to be applied consistently to all modules.

Reply:
I'm up for unifying names as `predef`.